MLBSpider.py

```diff
@@ -1,58 +0,0 @@
-import scrapy
-from datetime import datetime
-import datetime
-from scrapy_splash import SplashRequest
-from dateutil import tz
-import csv
-from scrapy.crawler import CrawlerProcess
-
-
-class MLBSpider(scrapy.Spider):
-    name = 'MLBSpider'
-    SPLASH_ARGS = {
-        'wait': 20.0,
-        'images': 0,
-        'render_all': 1,
-        'timeout': 40.0,
-        'resource_timeout': 40.0}
-
-    def start_requests(self):
-        url = 'http://www.espn.com/mlb/schedule'
-        yield SplashRequest(url, callback=self.parse, args=dict(self.SPLASH_ARGS))
-
-    def parse(self, response):
-
-        #Scrapes Home Team Name and appends to team list
-        teams = []
-        for team in response.xpath('//div[@class="responsive-table-wrap"][2]/table/tbody/tr/td[@class="home"]/div/a/abbr'):
-            team = team.xpath('@title').extract_first()
-            teams.append(team)

```

Comment:
Also, I _think_ you could replace all these `for team` loop lines with a list comprehension, something like
```
teams = [ team.xpath('@title').extract_first() for team in response.xpath(team_xpath) ]
```
List comprehensions are one of my favorite things about Python, and one of its most powerful features:
https://www.datacamp.com/community/tutorials/python-list-comprehension

Reply:
I originally shied away from using list comp on this one for fear of having lines way too long. It wasn't as bad as expected though and you're right that it's a million times better. Changed it.